ShareX.UploadersLib/FileUploaders/Lithiio.cs

```diff
@@ -77,7 +76,7 @@ public override UploadResult Upload(Stream stream, string fileName)
 
             if (result.IsSuccess)
             {
-                result.URL = result.Response;
+                result.URL = Config.UploadURL + result.Response;
```

Comment:
Tip: You can use `URLHelpers.CombineURL(Config.UploadURL, result.Response);` to combine urls more safely.